alerts/UA00001.json

```diff
@@ -0,0 +1,14 @@
+{
+  "dateRaised": 20200929,
+  "affectedFirmware": ["all"],
+  "hardwareLimited": [],
+  "description": "When using interrupt driven inputs if a hardware failure occurs that generates an extremely high interrupt rate (over 100,000 interrupts per second) the flight controller can become overloaded and cease to function. This affects the following types of inputs: RPM input with RPM_TYPE 1 or 2, Camera feedback with a CAM_FEEDBACK_PIN, Wheel encoders with WENC_TYPE=1, liquid fuel flow monitors with BATT_MONITOR=11 or 12, PWM rangefinder with RFND type 5 or 30, RSSI with RSSI_TYPE=4",
```

Comment:
RangeFinder type 22 is also affected type 5 is a compatability alias for type 22.

Reply:
And void `AP_Periph_FW::pwm_hardpoint_init()` looks like it installs a handler which could also be bad.

Also looks like we might also be able to save some bytes by using a `PWMSource` there.

Reply:
... and the AP_Radio backends?

... and RCOuput's serial-soft reader?

I *believe* anything doing an `attach_interrupt` may be affected....

Reply:
Added extra rangefinder and ESC telemetry